modal/runner/engines/vllm.py

```diff
@@ -116,7 +117,9 @@ async def generate(self, payload: CompletionPayload, params):
             print(f"Request completed: {throughput:.4f} tokens/s")
         except Exception as err:
             e = create_error_text(err)
-            print(e)
+            logging.exception(
+                "Failed generation", extra={"model": self.engine_args.model}
+            )
```

Comment:
opted to not convert the above `print(` to logging.info yet because in my test it did not show on modal (i think its because the default logger is set to WARN level, but we can followup with logging everywhere later. just a targeted change here for now)

Reply:
I hit the same thing, `logging.basicConfig(level=logging.INFO)` fixed it